resources/sql/backtick.sql

```diff
@@ -20,9 +20,9 @@ returning bq.*;
 -- name: queue-insert<!
 -- Insert a new job element
 insert into backtick_queue
-  (name, priority, state, tries, data, run_at, started_at, created_at, updated_at)
+  (name, priority, state, tries, data, started_at, created_at, updated_at)
 values
-  (:name, :priority, :state, :tries, :data, :run_at, now(), now(), now());
+  (:name, coalesce(:priority, now()), :state, :tries, :data, now(), now(), now());
```

Comment:
Do we want to set `started_at = now()` here? I see that's what was happening previously, but for a job queued for the future, `started_at` would equal queue insertion time rather than running time, making it redundant with `created_at`.